pkg/maps/maps.go

```diff
@@ -75,3 +75,11 @@ func Contains[M1, M2 ~map[K]V, K comparable, V comparable](superset M1, subset M
 func EqualFunc[M1 ~map[K]V1, M2 ~map[K]V2, K comparable, V1, V2 any](m1 M1, m2 M2, eq func(V1, V2) bool) bool {
 	return maps.EqualFunc(m1, m2, eq)
 }
+
+// Copy copies all key/value pairs in src adding them to dst.
+// When a key in src is already present in dst,
+// the value in dst will be overwritten by the value associated
+// with the key in src.
+func Copy[M1 ~map[K]V, M2 ~map[K]V, K comparable, V any](dst M1, src M2) {
```

Comment:
This will alter the original map, you can use `MergeCopy` in line 55.

Reply:
Ack, removed